adapters/onebot/src/bot/message.ts

```diff
@@ -141,8 +153,10 @@ export class OneBotMessenger extends Messenger<BaseBot> {
         await this.flush()
         this.stack.shift()
         await this.forward()
+      } else if ('id' in attrs) {
+        this.stack[0].author.messageId = attrs.id.toString()
       } else {
-        Object.assign(this.stack[0].author, pick(attrs, ['userId', 'username', 'nickname', 'avatar']))
+        Object.assign(this.stack[0].author, pick(attrs, ['userId', 'user-id', 'username', 'nickname', 'time']))
```

Comment:
`'userId', 'user-id'`?

Reply:
From document:
![image](https://user-images.githubusercontent.com/64234553/218254150-19a5c9cd-61a9-4343-a924-960a6a9e7b57.png)
The `user-id` is used here. Of course, for compatibility, I have kept `userId`.

Reply:
`user-id` will be transformed into `userId`

Reply:
But the issue now is that the user-id has not been transformed, do I need to handle this?

Reply:
You mean that there is only `user-id` in `attrs` instead of `userId`? If true, I think there is bugs in another place.

Reply:
yes, I found that there is only `user-id` and no `userId`. It never been transformed.

Reply:
Repro: param-case in the string form will be transformed, but JSX does not. Will be fixed on the @satorijs/element side.

Reply:
```suggestion
        Object.assign(this.stack[0].author, pick(attrs, ['userId', 'username', 'nickname', 'time']))
```

Reply:
> Repro: param-case in the string form will be transformed, but JSX does not. Will be fixed on the @satorijs/element side.

Should I open an issue for this?